app/assets/javascripts/discourse/app/templates/full-page-search.hbs

```diff
@@ -109,13 +109,22 @@
               @action={{action "toggleBulkSelect"}}
               class="btn-default bulk-select"
             />
-            {{#if this.selected}}
-              <DButton
-                @selected={{this.selected}}
-                @action={{action "showBulkActions"}}
-                @icon="wrench"
-                class="btn-default bulk-select-btn"
-              />
+            {{#if this.bulkSelectHelper.selected}}
+              {{#if this.useNewBulkActions}}
+                <TopicList::TopicBulkSelectDropdown
+                  @bulkSelectHelper={{this.bulkSelectHelper}}
+                  @afterBulkActionComplete={{action
+                    this.afterBulkActionComplete
+                  }}
```

Comment:
no need for the `action` helper
```suggestion
                  @afterBulkActionComplete={{
                    this.afterBulkActionComplete
                  }}
```

Reply:
Man this still trips me up so much, when to use vs. when not. I know in new gjs components we dont have to.

Reply:
The only (remaining) case where you need it is when the action doesn't have `@action` decorator and is inside an `actions: {}` object.